extensions/wikia/FacebookClient/FacebookClientController.class.php

```diff
@@ -146,23 +150,15 @@ public function disconnectFromFB() {
 		} else {
 			$this->status = 'error';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "error" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)